### ISSUE TITLE
Provide an example of how you would integrate it into a web app?

### ISSUE THREAD

**Issue by princemaple** [sikanhe/acme#5]
Hi,

Thanks for the great effort.

Could you provide an example of how you would integrate it into a web app?

**Comment by meatherly** [comment]
@princemaple Ha good question. I'm trying to figure that out myself. I'm new to all this encryption stuff. Have you found out anything since you've asked that question? I'm wondering if I only need to run the registration once or every time the app boots?

**Comment by princemaple** [comment]
Not really, I haven't looked into this since. I have other solutions, so this isn't a priority for me. Sorry mate.

**Comment by maartenvanvliet** [comment]
You'll only need to register once per letsencrypt account private key. You also only have to agree with the terms once. 

If you have the private key stored somewhere, you can re-initialize the Acme.Client and use it to authorize domains/request certificates, or other requests to the Acme server.
